fe/fe-core/src/main/java/com/starrocks/sql/analyzer/CreateTableAnalyzer.java

```diff
@@ -119,12 +119,21 @@ public static void analyze(CreateTableStmt statement, ConnectContext context) {
         statement.setCharsetName(analyzeCharsetName(statement.getCharsetName()).toLowerCase());
 
         KeysDesc keysDesc = statement.getKeysDesc();
+        List<ColumnDef> columnDefs = statement.getColumnDefs();
         if (statement.getSortKeys() != null) {
             if (keysDesc == null || keysDesc.getKeysType() != KeysType.PRIMARY_KEYS) {
```

Comment:
If you have already guaranteed the sort key is a subset of primarykey, under which instance, the sort key can be nullable?

Reply:
If not, why sort key must be non-nullable?